Source/diablo.h

```diff
@@ -2,6 +2,82 @@
 #ifndef __DIABLO_H__
 #define __DIABLO_H__
 
+#include "../types.h"
+
+#ifdef __cplusplus
+extern "C" {

```

Comment:
Do we know if this affects bin exact in any way?

Reply:
Looking at the travis result, it doesn't appear to make any change to the diff. In this case it shouldn't either, since it just moved some stuff from types.h to diablo.h.